website/content/docs/operations/stateful-workloads.mdx

```diff
@@ -0,0 +1,116 @@
+---
+layout: docs
+page_title: Considerations for Stateful Workloads
+description: |-
+  Learn about the available options for persistent workload storage on Nomad
+---
+
+# Considerations for Stateful Workloads
+
+By default, HashiCorp Nomad's allocation storage is ephemeral and can be discarded at any moment (such as a new deployment, rescheduling, or losing a Nomad client). This is undesirable when running persistent workloads such as databases.
```

Comment:
All the docs in the Nomad repo get hard-wrapped lines (this makes reviews easier on large paragraphs).

Reply:
Unresolving because the hard-wrap wasn't fixed.